api/ocm/extensions/download/handlers/helm/handler.go

```diff
@@ -101,41 +113,72 @@ func (h Handler) Download(p common.Printer, racc cpi.ResourceAccess, path string
 	return h.fromOCIArtifact(p, meth, path, fs)
 }
 
+// download downloads the chart and optional provenance file from an  oci.ArtifactAccess.
+// the format of the artifact is expected to match the official HELM Reference Specification
+//
+// see https://github.com/helm/community/blob/main/hips/hip-0006.md
```

Comment:
I am more of a fan of perma-links, but I guess that is debatable
```suggestion
// see https://github.com/helm/community/blob/dd5fe7878e293c573cc22db5d36558709c7b8a43/hips/hip-0006.md
```